content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -126,12 +135,12 @@ This frees you from writing your own API server to handle the custom resource,
 but the generic nature of the implementation means you have less flexibility than with
 [API server aggregation](#api-server-aggregation).
 
-Refer to the [Custom Controller example, which uses Custom Resources](https://github.com/kubernetes/sample-controller)
-for a demonstration of how to register a new custom resource, work with instances of your new resource type,
-and setup a controller to handle events.
+Refer to the [custom controller example](https://github.com/kubernetes/sample-controller)
+for an example of how to register a new custom resource, work with instances of your new resource type,
+and use a controller to handle events.
 
 {{< note >}}
-CRD is the successor to the deprecated *ThirdPartyResource* (TPR) API, and is available as of Kubernetes 1.7.
+CRD is the successor to the deprecated *ThirdPartyResource (TPR)* API.
```

Comment:
TPRs were removed in 1.9 (iirc). So they haven't been deprecated for a long time. :)